app/src/main/java/com/spendster/presentation/address/AddressActivity.java

```diff
@@ -1,15 +1,61 @@
 package com.spendster.presentation.address;
 
+import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
 
 import com.spendster.R;
 
-public class AddressActivity extends AppCompatActivity {
+public class AddressActivity extends AppCompatActivity implements AddressView {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_adress);
+
+        final EditText etFLName = findViewById(R.id.etFLName);
```

Comment:
@SimonMaha let's extract it into "initUI" method. Just to make functions smaller.